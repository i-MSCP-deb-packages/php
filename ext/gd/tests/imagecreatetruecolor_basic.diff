001+ 8363d3b388802a50a5f0e36beec28de3
001- 5a8fe9864cbd20e5dbe730c77f30db95