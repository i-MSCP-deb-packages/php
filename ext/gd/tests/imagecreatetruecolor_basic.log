
---- EXPECTED OUTPUT
5a8fe9864cbd20e5dbe730c77f30db95
---- ACTUAL OUTPUT
8363d3b388802a50a5f0e36beec28de3
---- FAILED
