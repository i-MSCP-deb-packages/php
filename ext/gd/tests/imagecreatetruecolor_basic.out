8363d3b388802a50a5f0e36beec28de3